contract/src/main/java/io/neow3j/contract/SmartContract.java

```diff
@@ -4,6 +4,7 @@
 import io.neow3j.constants.InteropServiceCode;
 import io.neow3j.constants.NeoConstants;
 import io.neow3j.contract.exceptions.UnexpectedReturnTypeException;
+import io.neow3j.crypto.ECKeyPair;
```

Comment:
Are these imports in the `SmartContract` required? Sorry, I haven't opened the pull request in my IntelliJ to check that out.

Reply:
You're right, I'll remove it.